content/learning-paths/smartphones-and-mobile/using_unity_machine_learning_agents/06-The-Unity-project.md

```diff
@@ -0,0 +1,183 @@
+---
+title: The Unity project
+weight: 6
+
+### FIXED, DO NOT MODIFY
+layout: learningpathall
+---
+
+## Prepare the project for training
+
+Before we move on to run the training, we will go through some of the key objects and scripts in Dr Arm. We will modify some of them to prepare for the training based on what we covered in the previous machine learning sections.
+
+## Overview of the project
+
+We will assume you still have the project open. From here onwards, you can ignore the "Ready to Play" version of the scene. We will use an incomplete scene to demonstrate some of the steps for using ML in Unity.
+
+1. Inside the _Project_ tab, navigate to _Assets->#DevSummit2022->Scenes_ (See Figure 1).
+
+   ![Project Scene Assets](images/project-assets.png "Figure 1. Project hierarchy showing our scene files in Assets/#DevSummit2022/Scenes")
+
+1. Double click the _Level DevSummit2022_ scene to open it.
+
+1. The hierarchy view lists the objects in the scene. You will see an object called AgentsSettings.
+
+1. Click the arrow next to AgentsSettings to reveal child objects Camera, ML-Player and ML-NPC.
+
+    The Camera object is the 3D camera - when played, the game will be rendered from this viewpoint.
+
+    The ML-Player object is our player-controlled character.
+
+    The ML-NPC object is our AI enemy character.
+
+1. Right click on _ML-Player_ and select _Align View to Selected_. This will change the scene view to the viewpoint of our player character at ground level.
+
+1. If you click on the arrow next to the object called UI, you will see several canvas objects. In Unity we use canvas objects to display the various parts of the user interface from the main menu (GamePlay_Canvas) to the touchscreen controls for mobile (MobileUI_Canvas).
```

Comment:
This line about UI seems unnecessary, we cover it below in better depth.
Indeed could more of this section be removed? Or maybe just slightly reworded - the align camera was largely just to get a good view of the scene so that people could see it was loaded in the tutorial, not sure it's strictly necessary?
You cover all the scene /hierarchy tab really well below, so this bit feels a little bit of a false start on that. (but some of it is needed/not duplicated!)